### PR TITLE
added some additional sources

### PR DESCRIPTION
**Pull request by rdmueller**
thanx for maintining this site! I thought it might be of interest to collect some more links to additional information / approaches regardings docs-as-code. I hope this is ok.

### CODE REVIEW

File: docs/guide/docs-as-code.rst [hunk 1]
```diff
@@ -53,3 +57,19 @@ and are gaining adoption in the writing community.
...
+
+The approach has been presented at several other conferences or just video casts by different speakers.
+
+**2017**
+
+* Software Architecture Summit 2017, Berlin, Germany: Gernot Starke, Ralf D. Müller: https://software-architecture-summit.de/softwarearchitektur/hitchhikers-guide-to-architecture-documentation/ (Half-Day-Workshop)
+
+**2018**
+
+* I'd rather be writing: Tom Johnson: https://www.youtube.com/watch?v=Z3e_38WS-2Q
```
Comment (plaindocs) [inline docs/guide/docs-as-code.rst]: Would you mind linkifying these?

**Review by plaindocs — CHANGES_REQUESTED** [review]
Thanks for this PR

Would probably be polite to mention that docToolchain is your own project, but looks like good reading.

File: docs/guide/docs-as-code.rst (hunk 1 above)
Comment (rdmueller) [reply docs/guide/docs-as-code.rst]: yes, sure. had to lookup the rst syntax, but I guess now it is fine, isn't it?

**Comment by rdmueller** [comment]
sorry for not directly mentioning that some if the links point to my own content :-|

**Comment by plaindocs** [comment]
Thanks! :+1:

**Review by plaindocs — APPROVED** [review]

**Pull request merged**